examples/userapi/05_conditionaldimension.ipynb

```diff
@@ -0,0 +1,729 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# ConditionalDimension tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial is designed to demonstrate how users can exploit Devito's ConditionalDimension in order to express loop structures that contain conditionals. It is known that Devito can generate a great variety of loop kernels using its Domain Specific Language. This tutorial introduces the user in building conditional loop structures through a series of examples. We show not only the DSL API used but the generated code as well.\n",
+    "\n",
+    "As in all our tutorials, we begin by specifying some parameters.\n",
+    "Our examples will be on 2D grids and we are going to update their values according to the specified conditionals."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Data([[1., 0., 0., 0., 0., 0., 0., 0., 0., 0.],\n",
+       "      [0., 1., 0., 0., 0., 0., 0., 0., 0., 0.],\n",
+       "      [0., 0., 1., 0., 0., 0., 0., 0., 0., 0.],\n",
+       "      [0., 0., 0., 1., 0., 0., 0., 0., 0., 0.],\n",
+       "      [0., 0., 0., 0., 1., 0., 0., 0., 0., 0.],\n",
+       "      [0., 0., 0., 0., 0., 1., 0., 0., 0., 0.],\n",
+       "      [0., 0., 0., 0., 0., 0., 1., 0., 0., 0.],\n",
+       "      [0., 0., 0., 0., 0., 0., 0., 1., 0., 0.],\n",
+       "      [0., 0., 0., 0., 0., 0., 0., 0., 1., 0.],\n",
+       "      [0., 0., 0., 0., 0., 0., 0., 0., 0., 1.]], dtype=float32)"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "\n",
+    "from devito import Dimension, Function, Grid\n",
+    "import numpy as np\n",
+    "\n",
+    "# We define a 10x10 grid, by default its dimensions are x, y\n",
```

Comment:
", by default its ..." -> drop, useless

Try to catch and rip off all the superfluous (sub-)comments. Don't forget this is supposed to be an advanced tutorial, and users should already be familiar with basic concepts such as Grid and Function